### PR TITLE
Add workflow to check the doc build status

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
- Update `version_updater` to handle `ci-docs.yml`.
- Update makefile to allow to override the variables: 
  - `SPHINXOPTS` 
  - `SPHINXBUILD` 
  - `PAPER` 
  - `BUILDDIR`
- Update `ci.yml` to dispatch the job for the docs.

### CODE REVIEW

File: .github/workflows/ci-docs.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          poetry --directory ./docs env info
+          poetry --directory ./docs install
+        timeout-minutes: 5
+
+      - name: Update translation
+        run: poetry run make --environment-overrides intl-fr
+        working-directory: docs
+        timeout-minutes: 2
+
+      - name: Check if `*.po` where modified
```
Comment (mmmarcos) [inline .github/workflows/ci-docs.yml:56]:
```suggestion
      - name: Check if `*.po` were modified
```

File: .github/workflows/ci-docs.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        id: modifed-po-file
```
Comment (mmmarcos) [inline .github/workflows/ci-docs.yml:57]: Typo?
```suggestion
        id: modified-po-file
```

**Review by mmmarcos — APPROVED** [review]
LGTM, just some typos to fix

File: .github/workflows/ci-docs.yml (hunk 2 above)
Comment (FirelightFlagboy) [reply .github/workflows/ci-docs.yml:57]: Inded + I don't use that id, so I've removed it.

**Pull request merged**